builtin/providers/aws/resource_aws_ami_from_instance_test.go

```diff
@@ -9,20 +9,22 @@ import (
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/awserr"
 	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/hashicorp/terraform/helper/acctest"
 	"github.com/hashicorp/terraform/helper/resource"
 	"github.com/hashicorp/terraform/terraform"
 )
 
 func TestAccAWSAMIFromInstance(t *testing.T) {
 	var amiId string
 	snapshots := []string{}
+	rInt := acctest.RandInt()
 
 	resource.Test(t, resource.TestCase{
 		PreCheck:  func() { testAccPreCheck(t) },
 		Providers: testAccProviders,
 		Steps: []resource.TestStep{
-			resource.TestStep{
-				Config: testAccAWSAMIFromInstanceConfig,
+			{
+				Config: fmt.Sprintf(testAccAWSAMIFromInstanceConfig, rInt),
```

Comment:
I usually wrap these in a function, merely to avoid test failures and confusions caused when reusing such configs and forgetting to pass the integer. Function with argument can enforce this at compilation time, which is nice.

With all that said it's probably a nitpick. :)